src/analysis-format.ts

```diff
@@ -383,3 +383,11 @@ export interface Slot extends Feature {
 
   // Something about fallback perhaps?
 }
+
+export interface Demo extends Feature {
```

Comment:
I guess that's not really a supported thing the way demos and jsdoc tags are treated right now.  Not extended anymore.